notification-api.md

```diff
@@ -156,84 +158,82 @@ that should be announced beyond the immediate effect of the primary action.
 
 ## Proposed Solution
 
-(This version inspired by [UIA Notification API](https://docs.microsoft.com/en-us/windows/win32/api/uiautomationcoreapi/nf-uiautomationcoreapi-uiaraisenotificationevent))
-
-For use cases that don't have an express UI tie-in, it makes sense to provide a solution
-not expressly tied to HTML element, but for which other document-centric information can
-be inferred (such as language).
+We provide an API on Documents and Elements to enable posting imperative notifications with 
+independently-managable queues. Additionally, Element-based context should be assumed for
+language.
 
 ```js
-// As the assistive technology to notify the user, given a specific string
-document.ariaNotify( "Selected text is now glowing blue." );
+// Queue a message to the body element's notification queue given the provided string
+document.body.ariaNotify( "Selected text is now glowing blue." );
 ```
+
 A screen reader or other assistive technology tool would speak or show "selected text is now
 glowing blue". For users without assistive technology tools running, nothing would happen.
 The call to the API has no web-observable side effects and its use should not infer that the 
 user is using assistive technology. 
 
-Here the content author provides a label to group a set of related notification about the 
-clipboard.
+The API can also be called on the document. The document also has a single [indpendent] queue
```

Comment:
[indpendent] 

spelling, and perhaps use parentheses instead?

Reply:
👍